open-apparel-registry/open-apparel-registry
# Add Open Graph preview image

## Overview

Adds a preview image for Open Graph.

Connects #2283 

## Demi

![Screen Shot 2022-10-31 at 11 00 10 AM](https://user-images.githubusercontent.com/8356789/199054176-5fbdc5c7-d3e2-4af2-9de2-84a3da2a9abe.png)

## Notes

I added the twitter image tag, but the other twitter tags are missing. Perhaps they should be added?

## Testing Instructions

- Create an account for and install `ngrok`.
- `ngrok http 6543`
- Paste the generated URL into Slack
- Ensure the banner shows up

## Checklist
